oak_containers_hello_world_untrusted_app/tests/integration_test.rs

```diff
@@ -19,32 +19,49 @@ use std::sync::Once;
 
 use oak_containers_launcher::Args;
 use oak_crypto::encryptor::ClientEncryptor;
+use once_cell::sync::Lazy;
 
 const EMPTY_ASSOCIATED_DATA: &[u8] = b"";
 
-static INIT: Once = Once::new();
+static INIT_LOGGING: Once = Once::new();
 
 fn init_logging() {
-    INIT.call_once(|| {
+    INIT_LOGGING.call_once(|| {
         env_logger::init();
     });
 }
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 4)]
-async fn hello_world() {
+static INIT_DEPENDENCIES: Lazy<anyhow::Result<()>> = Lazy::new(|| {
```

Comment:
I am not sure this is working as intended. At least on my machine (`cargo nextest run --package=oak_containers_hello_world_untrusted_app`), if I run the tests in this package, the dependencies get built twice in parallel (which fails because of some race condition in the output files). Not sure why though, maybe the tests are running on different processes and not sharing the lock?

cc @conradgrobler @ipetr0v

Reply:
@andrisaar 

indeed, nextest runs them separately, which means each of them will still try to rebuild all the dependencies: https://nexte.st/book/usage.html?highlight=process#limitations

Reply:
https://github.com/project-oak/oak/pull/4687